kernel/kernel-api/src/main/java/io/delta/kernel/internal/TransactionImpl.java

```diff
@@ -486,6 +479,30 @@ private long doCommit(
     }
   }
 
+  private void incrementMetricsForFileActionRow(Row fileActionRow, TransactionMetrics txnMetrics) {
```

Comment:
nit: In general, the object being acted upon should come before the object doing the acting.

i.e. we are applying the fileActionRow (i.e. extracting metrics from it) and then applying that information to the txnMetrics, then the param order should be `txnMetrics, fileActionRow`

Reply:
And we should have a kernel java style guide that lists these preferences :D